src/vs/workbench/parts/extensions/browser/extensionsActions.ts

```diff
@@ -78,6 +80,14 @@ export class InstallAction extends Action {
 	}
 
 	run(): TPromise<any> {
+		if (this.extension.recommendationInfo) {
+			this.telemetryService.publicLog('extensionRecommendations:install', {
```

Comment:
Unless you are interested only in the intent to install, you will likely get events for installs which fail or are cancelled.